networking/external_dns_operator/external-dns-operator-release-notes.adoc

```diff
@@ -18,10 +18,23 @@ These release notes track the development of the External DNS Operator in {produ
 [id="external-dns-operator-release-notes-1.3.0"]
 == External DNS Operator 1.3.0
 
-This update includes a rebase to the 0.14.2 version of the upstream project.
+The following advisory is available for the External DNS Operator version 1.3.0:
+
+*link:https://access.redhat.com/errata/RHEA-2024:8550[RHEA-2024:8550 Produce Enhancement Advisory]
+
+This update includes a rebase to the 0.14.2 version of the upstream project:
 
 * link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/[Kubernetes 0.14.2 Information]
```

Comment:
This link still opens to wrong page `https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/`

Reply:
Right. I don't think we need a link to any upstream at all. We try to stay away from referencing upstream as it has a broader set of features from what we support.

Reply:
okay, will remove link.